snc.sh

```diff
@@ -158,6 +167,10 @@ ${YQ} write --inplace $INSTALL_DIR/manifests/cluster-ingress-02-config.yml spec[
 # Start the cluster with 10GB memory and 4 CPU create and wait till it finish
 export TF_VAR_libvirt_master_memory=10192
 export TF_VAR_libvirt_master_vcpu=4
+
+# Add codeReadyContainer as invoker to identify it with telemeter
+export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
```

Comment:
:+1:

Reply:
Any idea which case is typical for the invoker ("CodeReadyContainers" VS "codeReadyContainers", or even with a space "CodeReady Containers"?)

Reply:
@cfergeau whatever easy for us to use I mentioned `codeReadyContainers` to original issue and didn't see any push back so went with it :)